src/endpoint.js

```diff
@@ -19,12 +19,14 @@ module.exports = class AbstractEndpoint {
     this.isPaginated = false
     this.maxPageSize = 200
     this.isBulk = false
-    this.supportsBulkAll = true
+    this.supportsBulkAll = true 
```

Comment:
Remove the whitespace (the linter will complain about this too if you run `npm run lint` / `yarn lint`)